<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>all_drugs_form</class>
 <widget class="QFrame" name="all_drugs_form">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="3">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Всего лс на сумму 0.00р</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="QTableWidget" name="tableWidget"/>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="new_drug_btn">
       <property name="text">
        <string>Новое лс</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="close_btn">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="del_drug_btn">
       <property name="text">
        <string>Удалить лс</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>close_btn</sender>
   <signal>clicked()</signal>
   <receiver>all_drugs_form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>628</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>668</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
